x-pack/test/security_solution_cypress/cypress/e2e/investigations/timelines/unsaved_timeline.cy.ts

```diff
@@ -146,8 +141,11 @@ describe.skip('Save Timeline Prompts', { tags: ['@ess'] }, () => {
 
 // In serverless it is not possible to use the navigation menu without closing the timeline
 describe('Save Timeline Prompts', { tags: ['@serverless'] }, () => {
```

Comment:
what do you think about having `serverless` in the title, something like `Save timeline prompts - serverless`? Right now both describe in this file have the exact same title...